application/modules/admin/views/scripts/shipment/ship-it.phtml

```diff
@@ -17,8 +17,14 @@
             ?>
                     <li><i class="fa fa-minus"></i>
                         <label>
-                            <input id="node-<?php echo $ckey; ?>" data-id="custom-<?php echo $ckey; ?>" type="checkbox"  <?php if(count($unenrolledParticipants) == 0) {  ?> checked="true" <?php  } ?>>
+                        <input id="node-<?php echo $ckey; ?>" data-id="custom-<?php echo $ckey; ?>" type="checkbox"  <?php if(count($unenrolledParticipants) == 0) {  ?> checked="true" <?php  } ?>>
                             <?php echo $ckey; ?>
+                        <input type="text" name="country_last_dates[<?php  echo $ckey; ?>]" size="50" maxlength="45" 
+                            value="<?php  echo $this->shipment->lastdate_response;?>"
+                            readonly="readonly"
+                           class="input-sm datepicker ml-auto"
```

Comment:
This should actually be a free text field rather than a date picker. The administrator should be able to enter any free text into this field.